mindmeld/components/nlp.py

```diff
@@ -421,10 +422,11 @@ def __init__(self, app_path, resource_loader=None, config=None):
         self.name = app_path
         self._load_custom_features()
         self.domain_classifier = DomainClassifier(self.resource_loader)
+        self.progress_bar = progress_bar or tqdm(total=0, desc="Classifier progress")
```

Comment:
Do we have a single progress bar for the entire training process, or an individual progress bar for each domain?

Reply:
One progress bar for all the classifiers

Reply:
Should we have an option to disable the progress bar?